src/state_pred/constants.py

```diff
@@ -5,29 +5,29 @@
 
 # PHYSICAL PROPERTIES
 
-DIST_REAR_AXEL = 0.85   # m
+DIST_REAR_AXEL = 0.85  # m
 DIST_FRONT_AXEL = 0.85  # m
 WHEEL_BASE = DIST_REAR_AXEL + DIST_FRONT_AXEL
 
-MASS = 15   # kg
+MASS = 15  # kg
 
 CORNERING_STIFF_FRONT = 16.0 * 2.0  # N/rad
-CORNERING_STIFF_REAR = 17.0 * 2.0   # N/rad
+CORNERING_STIFF_REAR = 17.0 * 2.0  # N/rad
 
-YAW_INERTIA = 22    # kg/(m^2)
+YAW_INERTIA = 22  # kg/(m^2)
 
 # INPUT CONSTRAINTS
 
-SPEED_MIN = 2       # m/s
-SPEED_MAX = 10      # m/s
+SPEED_MIN = 1  # m/s
+SPEED_MAX = 100  # m/s
 
-STEER_ANGLE_MAX = np.radians(37)    # rad
-STEER_ANGLE_DELTA = np.radians(1)   # rad
+STEER_ANGLE_MAX = np.radians(90)  # rad
```

Comment:
I'm curious why you changed these constants, especially acceleration_min because I thought that was how we would be braking?

Reply:
lowkey idk why these were changed... that's weird.